rust/blockstore/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 pub mod types;
 
 pub mod arrow;
+#[cfg(test)]
+mod blockfile_writer_test;
```

Comment:
i think using a test/ folder inside the crate will let you avoid having to export this

https://doc.rust-lang.org/cargo/guide/project-layout.html

Reply:
👍 will move in a future PR